packages/core/src/components/scrollable/scrollable.scss

```diff
@@ -1,32 +1,32 @@
 @use "~dso-toolkit/src/utilities";
 @use "~dso-toolkit/src/variables/units";
+@use "~dso-toolkit/src/variables/media-query-breakpoints";
 
 :host {
-  display: block;
-  height: 100%;
+  display: flex;
+  flex: 1;
+  min-height: 0;
```

Comment:
Waarom is dit nodig? alles heeft in principe een min-height 0, tenzij anders bepaald. het lijkt me een reset, maar ik zie niet wat er gereset wordt ;)